delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/indexing/BlazegraphGlobalEventLog.scala

```diff
@@ -0,0 +1,76 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.blazegraph.indexing
+
+import akka.persistence.query.Offset
+import cats.implicits._
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.ExpandedJsonLd
+import ch.epfl.bluebrain.nexus.delta.sdk.eventlog.{EventExchangeCollection, GlobalEventLog}
+import ch.epfl.bluebrain.nexus.delta.sdk.model._
+import ch.epfl.bluebrain.nexus.delta.sdk.model.organizations.OrganizationRejection.OrganizationNotFound
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectRef
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectRejection.ProjectNotFound
+import ch.epfl.bluebrain.nexus.delta.sdk.{Organizations, Projects}
+import ch.epfl.bluebrain.nexus.sourcing.EventLog
+import ch.epfl.bluebrain.nexus.sourcing.projections.ProjectionStream._
+import ch.epfl.bluebrain.nexus.sourcing.projections.{Message, ProjectionId}
+import fs2.Stream
+import monix.bio.{IO, Task}
+import monix.execution.Scheduler
+
+/**
+  * An implementation of [[GlobalEventLog]]
+  */
+final class BlazegraphGlobalEventLog private (
+    eventLog: EventLog[Envelope[Event]],
+    projects: Projects,
+    orgs: Organizations,
+    eventExchanges: EventExchangeCollection
+)(implicit projectionId: ProjectionId, sc: Scheduler)
+    extends GlobalEventLog[Message[ResourceF[ExpandedJsonLd]]] {
```

Comment:
For Blazegraph indexing, we actually care more about NTriples no ?
It could be NTriples directly here ?

Reply:
It needs to be `ExpandedJsonLd` or `Graph` because we need to be able to add metadata triples.

Reply:
No. We need the graph to add some triples to it manuaally